classes/form/policydoc.php

```diff
@@ -90,8 +90,8 @@ public function definition() {
         $mform->addRule('content_editor', null, 'required', null, 'client');
 
         if (!$formdata->id || $formdata->status == policy_version::STATUS_DRAFT) {
-            // Creating a new version or editing a draft.
-            $mform->addElement('hidden', 'minorchange', 1);
```

Comment:
So how/where is the value of this hidden element set now (after removing the third param)?

Reply:
it's part of $formdata , see line 130 of the same file and line 157 of the editpolicydoc.php

Reply:
Oh I see now. Thanks.